fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -47,6 +47,14 @@ def take_off
           Fastlane::CommandsGenerator.start
         end
       end
+
+      # We need custom treatment for `pem`
+      def module_name_for_tool(tool_name)
+        if tool_name == "pem"
+          return PEM
+        end
+        return tool_name.fastlane_class
```

Comment:
Is this code correct as written? It feels like the if-branch returns a Module object, but the base case returns a String